VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/Attributes/OnPropertyChangedWithoutCallerMemberName/OnPropertyChangedWithoutCallerMemberNameCodeFix.cs

```diff
@@ -51,9 +51,9 @@ public override ImmutableArray<string> FixableDiagnosticIds
 
             editor.ReplaceNode(param, newParam);
 
-            var parentClass = methodDeclaration.Ancestors().OfType<ClassDeclarationSyntax>().FirstOrDefault();
+            var parentNode = GetParent(methodDeclaration);
```

Comment:
Don't duplicate the helper -- move it out to the extensions. Perhaps something like `public SyntaxNode GetEnclosingTypeDeclaration(this SyntaxNode node)`?